docs/markdown/snippets/introspect_targets_subproject.md

```diff
@@ -0,0 +1,3 @@
+## Added subproject name to --targets introspection
+
+This allows association of targets with subprojects
```

Comment:
Instead of putting this in a separate snippet, maybe merge this with [this file](https://github.com/mesonbuild/meson/blob/master/docs/markdown/snippets/introspect_multiple.md) by adding a new bullet point at the end?

Reply:
yes that's a good idea.